### ISSUE TITLE
New release 2.3.0 doesn't compile

### ISSUE THREAD

**Issue by lehors** [cgkineo/adapt-pageNav#46]
### Subject of the issue/enhancement/features

Using the latest release my build fails with the following error:
```
Fatal error: NameError: error evaluating function `darken`: variable @btn-item-color is undefined in src/components/adapt-pageNav/less/pageNav.less on line 33, column 25:
32     &[aria-current="page"] {
33       background-color: darken(@btn-item-color, 20%);
34     }

```
Installing 2.2.1 solves the problem.

### Your environment
ubuntu 22.04.2
node v14
authoring tool v0.11.1
framework v5.31.11

**Comment by swashbuck** [comment]
Caused by #32

**Comment by swashbuck** [comment]
Fix pending on #47
